warehouse/templates/search/results.html

```diff
@@ -35,9 +35,6 @@ <h3 class="package-snippet__title">
 
```

Comment:
@bstutes Making a new PR shouldn't be necessary, you should be able to just push a new commit to your existing branch.